File: test-artifacts/p-visitor-js/function-invocation-replacer/int88888888_015_in.js
function f_gold(num) {
    if (num < 0) {
        const retval_1 = f_gold(-num);
        return retval_1;
    }
    if (num === 0 || num === 7) {
        return true;
    }
    if (num < 10) {
        return false;
    }
    const retval_2 = f_gold(Math.floor(num / 10) - 2 * (num % 10));
    return retval_2;
}

File: test-artifacts/p-visitor-py/function-invocation-replacer/int88888888_015_in.py
def f_gold(num):
    if num < 0:
        retval_1 = f_gold(-num)
        return retval_1
    if num == 0 or num == 7:
        return True
    if num < 10:
        return False
    retval_2 = f_gold(num / 10 - 2 * (num - num / 10 * 10))
    return retval_2